docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project.
+
+.Prerequisites
+
+* Projects imported into VS Code.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_mta_ex_icon.png[MTA code extension]) on the Activity bar.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Complete the following configuration fields:
+
+* *Input*: Click *Add* and either:
+
+** Enter the input file or directory and press Enter.
```

Comment:
Are you sure this is a keyboard command? Not a button? In either case, "Enter" needs to be marked up. My guess is that the user has to click a button with a mouse. See IBM.